src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleWidget.ts

```diff
@@ -80,7 +80,6 @@ export abstract class TerminalAccessibleWidget extends DisposableStore {
 			quickSuggestions: false,
 			renderWhitespace: 'none',
 			dropIntoEditor: { enabled: true },
-			accessibilitySupport: this._configurationService.getValue<'auto' | 'off' | 'on'>('editor.accessibilitySupport'),
 			cursorBlinking: this._configurationService.getValue('terminal.integrated.cursorBlinking'),
```

Comment:
Since this is now overlaid in the middle of the workbench we should use the editor's value

Reply:
I just noticed that by default, the terminal cursor doesn't blink, and it's a boolean setting. Therefore, it acts differently than the editor cursor blinking setting and I'm wondering whether we should just keep it as-is, then, or change it so that if it is enabled, it uses the editor value.

Reply:
I think just go with the editor's